hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledExecutorServiceProxy.java

```diff
@@ -345,6 +320,60 @@ public void shutdown() {
         waitWithDeadline(calls, SHUTDOWN_TIMEOUT, TimeUnit.SECONDS, WHILE_SHUTDOWN_EXCEPTION_HANDLER);
     }
 
+    private <V> void retrieveAllMemberOwnedScheduled(Map<Member, List<IScheduledFuture<V>>> accumulator) {
+        try {
+            InvokeOnMembers invokeOnMembers = new InvokeOnMembers(getNodeEngine(), getServiceName(),
+                    new GetAllScheduledOnMemberOperationFactory(name),
+                    getNodeEngine().getClusterService().getMembers());
```

Comment:
Yes, @jerrinot brought this up also, but this is consistent with the ExecutorService. This is a computation service, being an executor and handling tasks and such, however, the members (including lite) do store some data, the task definition in this case.